crypto/evp/evp_locl.h

```diff
@@ -108,6 +108,22 @@ struct evp_keyexch_st {
     OSSL_OP_keyexch_set_params_fn *set_params;
 } /* EVP_KEYEXCH */;
 
+struct evp_signature_st {
+    char *name;
+    OSSL_PROVIDER *prov;
+    CRYPTO_REF_COUNT refcnt;
+    CRYPTO_RWLOCK *lock;
+
+    EVP_KEYMGMT *keymgmt;
+
+    OSSL_OP_signature_newctx_fn *newctx;
+    OSSL_OP_signature_sign_init_fn *sign_init;
+    OSSL_OP_signature_sign_fn *sign;
```

Comment:
I'm surprised there aren't any verification functions.  After all, the name `EVP_SIGNATURE` suggests that this method operates on signatures, so it would be *weird* if verification functionality is on another type.  Can I assume that it's upcoming functionality in this method?

Reply:
Yes, I expect verification to be part of the same operation. I had been focusing on the sign aspects - but I should probably add verify to this PR too.

Reply:
Fixed